src/Symfony/Bundle/FrameworkBundle/Validator/ConstraintValidatorFactory.php

```diff
@@ -61,13 +62,18 @@ public function __construct(ContainerInterface $container, array $validators = a
      *
      * @return ConstraintValidatorInterface A validator for the supplied constraint
      *
-     * @throws UnexpectedTypeException When the validator is not an instance of ConstraintValidatorInterface
+     * @throws InvalidArgumentException When the validator class does not exist
+     * @throws UnexpectedTypeException  When the validator is not an instance of ConstraintValidatorInterface
      */
     public function getInstance(Constraint $constraint)
     {
         $name = $constraint->validatedBy();
 
         if (!isset($this->validators[$name])) {
+            if (!class_exists($name)) {
+                throw new InvalidArgumentException(sprintf('Constraint validator class "%s" does not exist.', $name));
```

Comment:
Thanks, agree with you. 

Now, what do you think about the message ? Should be a little bit friendly ?